teamscale-gradle-plugin/src/main/kotlin/com/teamscale/TeamscalePlugin.kt

```diff
@@ -119,6 +119,7 @@ open class TeamscalePlugin : Plugin<Project> {
             }
             teamscaleReportTask.uploadTask = teamscaleUploadTask
             teamscaleUploadTask.reports.add(extension.report.getReport())
+            teamscaleUploadTask.mustRunAfter(teamscaleReportTask)
```

Comment:
That's an odd change. Is this related to something described in the issue description? Or did you just notice that this would make the plugin more resilient?

Reply:
I moved that part over form the TIA demo repository as it should IMO behave like this out of the box.